plugins/playlist/src/components/PlaylistEditDialog/PlaylistEditDialog.tsx

```diff
@@ -74,13 +75,33 @@ export const PlaylistEditDialog = ({
 }: PlaylistEditDialogProps) => {
   const classes = useStyles();
   const identityApi = useApi(identityApiRef);
-
+  const playlistApi = useApi(playlistApiRef);
+  const playlistPromise = useRef(
+    playlistApi.getAllPlaylists({ editable: true }),
```

Comment:
We probably don't want to limit this to `editable` playlists - this would only include playlists the user has permissions to modify and not every playlist the user has permissions to `read` so duplicates could still appear in the list of every playlists visible to the user 
```suggestion
    playlistApi.getAllPlaylists(),
```

Reply:
Typescript build failed 
after  playlistApi.getAllPlaylists(), so passed empty object to resolve is 
 playlistApi.getAllPlaylists({}),